api/v1alpha1/driver_types.go

```diff
@@ -110,14 +110,14 @@ type ProvisionerResourcesSpec struct {
 	Plugin        *corev1.ResourceRequirements `json:"plugin,omitempty"`
 }
 
-type ProvisionerSpec struct {
+type ControllerPluginSpc struct {
```

Comment:
ControllerPluginSpc to ControllerPluginSpec

Reply:
done